### PR TITLE
Week 17 - First API - Fatima Gamero Romero

### PR DESCRIPTION
**Pull request by Fatima-GR**

### CODE REVIEW

**Comment by Fatima-GR** [comment]
https://booksfirstapi.herokuapp.com/

File: server.js [hunk 1]
```diff
@@ -1,33 +1,140 @@
...
+
+//  --------- SIMPLE ENDPOINTS --------------//
+// Only use the response parameter as it gets a general response
+// Therefore, we don't need the request parameter to ask the user for a specific value.
+
+// Endpoint 1:  Front-end get just a sentence (http://localhost:8080)
+app.get("/", (req, res) => {
+  res.send(
+    "My name is Fatima and if you know me well, you must already know that I just love cats!"
+  );
```
Comment (KaraHowes) [inline server.js:24]: I think here, in your welcome message, you could write "add "/endpoints" in browser to view all endpoints", just to give the user a clearer idea how to use the API

File: server.js [hunk 2]
```diff
@@ -1,33 +1,140 @@
...
+});
+
+//Endpoint 3: Front-end get an array of objects that holds info about books: (http://localhost:8080/books)
+app.get("/books", (req, res) => {
+  res.json(booksData);
+});
+
+
+
+// ---------   ENDPOINTS USING PATH PARAMS -----------//
```
Comment (KaraHowes) [inline server.js:39]: I really like how you have separated your different endpoints. It shows a clear understanding of what you have done and why

**Review by KaraHowes — COMMENTED** [review]
I love this project, Fatima!
Your code is clear, well-commented, consistent and extremely easy-to-follow. It is clear that you really understand exactly what you have done and, most importantly, why. I honestly can not make suggestions to improve the code-itself! Congratulations on a fantastic project!

**Pull request closed without merging**